css/app.css

```diff
@@ -39,7 +39,7 @@ body {
     height: 100%;
     width: 100%;
     z-index: -10;
-    margin-left:400px;
+    /*margin-left:400px;*/
```

Comment:
When calculating the width of the `#content` div to 100%, it was going to the width of the entire screen. When I was adding the margin, It had already generated to be the width of the screen, so it was pushing it over and in turn, pushing 400px of the map that had already been created out of the window. That's why everything was off-center and the point wasn't being displayed on the map.